services/community-site/src/utils/claim.ts

```diff
@@ -0,0 +1,80 @@
+import { ethers } from 'ethers';
```

Comment:
I wouldn't add it to utils. Maybe create a new service for this?

Reply:
Done 👍🏼